package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ytdl-process",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Was this intentional?

Reply:
Yeah, we are changing the interface so we should be bumping the version number shouldn't we?

Reply:
fixed